src/main/java/com/twilio/oai/api/PhpApiResourceBuilder.java

```diff
@@ -138,11 +163,70 @@ private void updateNamespaceSubPart(CodegenOperation codegenOperation) {
 
     @Override
     public ApiResourceBuilder updateOperations(Resolver<CodegenParameter> codegenParameterIResolver) {
-        ApiResourceBuilder apiResourceBuilder = super.updateOperations(codegenParameterIResolver);
-        reorderParams(apiResourceBuilder);
-        this.addOptionFileParams(apiResourceBuilder);
+        PhpJsonRequestBodyResolver jsonRequestBodyResolver = new PhpJsonRequestBodyResolver(this, codegenPropertyIResolver);
+        this.codegenOperationList.forEach(co -> {
+            updateNestedContent(co);
+            List<String> filePathArray = new ArrayList<>(Arrays.asList(co.baseName.split(PATH_SEPARATOR_PLACEHOLDER)));
+            String resourceName = filePathArray.remove(filePathArray.size()-1);
+
+            jsonRequestBodyResolver.setResourceName(resourceName);
+            co.allParams.stream()
+                    .forEach(item -> {
+                        CodegenModel model = getModel(item.dataType);
+                        // currently supporting required and conditional parameters only for request body object
+                        if (model != null) {
+                            model.vendorExtensions.put("x-constructor-required", true);
+                            jsonRequestBodyResolver.resolve(item, codegenParameterIResolver);
+                        }
+                    });
+            co.pathParams = co.pathParams.stream()
```

Comment:
Do we need this, since this code is already written in parent class ?

Reply:
yes, in some cases it is not able to resolve the below params.